examples/compilation/library.jl

```diff
@@ -0,0 +1,102 @@
+# EXCLUDE FROM TESTING
+
+using CUDArt
```

Comment:
I'm thinking this should be incorporated into CUDArt and a `@compile` macro should be exported?